flepimop/gempyor_pkg/src/gempyor/initial_conditions.py

```diff
@@ -99,7 +99,7 @@ def get_from_config(self, sim_id: int, modinf) -> np.ndarray:
                 allow_missing_subpops=self.allow_missing_subpops,
             )
         else:
-            raise NotImplementedError(f"unknown initial conditions method [got: {method}]")
+            raise NotImplementedError(f"Unknown initial conditions method [recieved: {method}].")
```

Comment:
```suggestion
            raise NotImplementedError(f"Unknown initial conditions method [received: {method}].")
```

probably ctrl+f recieve to ensure not making this error anywhere else.